src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -398,6 +398,7 @@ public static class Error
          public static string DistributionUnknown(string distribution) => $"Distribution '{distribution}' is unknown.";
          public const string NameIsRequired = "Name is required.";
          public const string MoleculeIsRequired = "Molecule is required.";
+         public const string CategoryIsRequired = "Category is required.";
```

Comment:
@Yuri05 @PavelBal I am not sure if Category is the right name. For instance CYP3A4-Healthy, healthy is the category

![image](https://user-images.githubusercontent.com/1041237/140411754-538f9a6d-da32-47b3-a160-efc517557859.png)


Also not sure that Settings is the right name :)

Reply:
Category --> Phenotype? Sub-Population?
Settings --> Reference Population (--> move "Enzyme" to "Properties")
Properties --> Global Enzyme Properties

Reply:
Enzyme can't be move (at least not easily) to Properties as the "views" below will also be used in the Individual